### PR TITLE
[risk=no][no ticket] stick thing in componentDidUpdate

### PR DESCRIPTION
**Pull request by als364**
Description:

---
**PR checklist**

- [ ] This PR meets the Acceptance Criteria in the JIRA story
- [ ] The JIRA story has been moved to Dev Review
- [ ] This PR includes appropriate unit tests
- [ ] I have run and tested this change locally
- [ ] I have run the E2E tests on ths change against my local UI and/or API server with `yarn test-local` or [yarn test-local-devup](https://github.com/all-of-us/workbench/blob/master/e2e/README.md#examples)
- [ ] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer
- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and notified API consumers
- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later

### CODE REVIEW

File: ui/src/app/pages/admin/admin-institution-edit.tsx [hunk 1]
```diff
@@ -164,6 +164,20 @@ export const AdminInstitutionEdit = fp.flow(withUrlParams(), withNavigation)(cla
     }
   }
 
+  async componentDidUpdate(prevProps: Props) {
+    // If institution short Name is passed in the URL get the institution details
+    if (hasNewValidProps(this.props, prevProps, [p => p.urlParams.institutionId])) {
+      const loadedInstitution = await institutionApi().getInstitution(this.props.urlParams.institutionId);
```
Comment (jmthibault79) [inline ui/src/app/pages/admin/admin-institution-edit.tsx:170]: since this block duplicates one in componentDidMount, we can extract a function

**Review by jmthibault79 — APPROVED** [review]
Works for me locally.

I don't understand React well enough to comment on didMount vs didUpdate, unfortunately

File: ui/src/app/pages/admin/admin-institution-edit.tsx (hunk 1 above)
Comment (als364) [reply ui/src/app/pages/admin/admin-institution-edit.tsx:170]: this is true but also I'm going to delete it in #5450 so I don't care very muh

**Comment by als364** [comment]
puppeteer error seems transient / unrelated, merging

**Pull request merged**